src/code/cutscene_camera.c

```diff
@@ -100,19 +104,19 @@ u8 CutsceneCamera_ProcessSpline(CutsceneCamera* csCamera) {
         sp5C = false;
```

Comment:
Maybe sp5C could be something like `finishedCurrentSpline`? Or just `finishedSpline`? Based on the code above I mean, which Github won't let me highlight for some reason.

Reply:
You mean:
```c
if (csCamera->eyeCmd[csCamera->atInterp.curPoint].interpType <
    csCamera->atCmd[csCamera->eyeInterp.curPoint].interpType) {
    sp5C = false;
}
```
This one is interesting, I don't think this has to do with whether the spline is finished? Or maybe it does because `CS_CAM_INTERP_OFF` is the highest enum value (7), is that what you were thinking? Other than than, both the eye points and the at points can have separate interpolations, so this is checking if the eye interpolation is a lower order than the at interpolation? Not sure I understand the purpose of this